client/app/telecom/telecom.less

```diff
@@ -34,3 +34,7 @@
 .overflow-visible {
     overflow: visible !important;
 }
+
+.oui-header-tabs {
```

Comment:
Maybe this should be managed by oui-kit

Reply:
No, because this style concerns Telecom Universe, i'm not sure all used tabs want this rule in all OVH Sites

Reply:
There shouldn't be style specific to a universe, all the managers should be unified

Reply:
Yes I agree, the reason of this code is related to this component (editable-service-name)
https://github.com/ovh-ux/ovh-manager-telecom/blob/master/client/app/telecom/telephony/alias/telecom-telephony-alias-main.view.html#L13-L18

This brings some issues to fit with ui-kit `oui-header-tabs`. Once we'll remove this component, I'll delete this custom style (and use the proper header component into client/app/telecom/telephony/alias/telecom-telephony-alias-main.view.html)

Reply:
> (…) Once we'll remove this component, I'll delete this custom style (…)

It would be perfect to add this as a comment `TODO: …`.